scripts/show_store_details.py

```
@@ -0,0 +1,122 @@
+#!/usr/bin/env python3
+"""
+Скрипт для просмотра детальной информации о Google File Search Store
+и его документах (батчах с catalog записями).
+
+Использование:
+    python scripts/show_store_details.py
+"""
+import os
+import sys
```

Flake8 (7.3.0)

[error] 'sys' imported but unused

(F401)